instagram/src/components/Comments/CommentSection.js

```diff
@@ -38,7 +38,7 @@ class CommentSection extends Component {
                 <input type="submit" value="Say what'chu gon' say, bruh."/>
             </form>
             {/* breakdown array of comments and pass each item in to <Comment/> */}
-            {this.props.comments.map(comm => <Comment comments={comm}/>)}
+            {this.state.comments.map(comm => <Comment comments={comm}/>)}
```

Comment:
ok variable names and good use of map for the variables it might be worth using full name and keep them fairly verbose though